packages/amplify-graphql-migration-tests/src/__tests__/graphql-version-migration.test.ts

```diff
@@ -25,7 +25,7 @@ describe('v1 to v2 migration', () => {
       const v1nestedStackNames = Object.keys(v1result.stacks).filter((stackName) => stackName !== 'ConnectionStack'); // The v1 transformer puts all connection resolvers in a 'ConnectionStack'. This stack does not define any data resources
 
       // verify root stack diff
-      const diff = cdkDiff.diffTemplate(v1result.rootStack, v2result.rootStack);
```

Comment:
Breaking change with version upgrade: https://github.com/aws/aws-cdk/pull/28336

`diffTemplate` is no longer exported, but `fullDiff` seems to just be a wrapper.